src/drivers/ports/esp32/can_port.i

```diff
@@ -139,33 +216,53 @@ static void isr(void *arg_p)
 
     regs_p = self_p->dev_p->regs_p;
 
-    /* Read the interrupt status. */
+    /* Read the interrupt status and status registers. */
```

Comment:
Remove `` and status registers``. I added it when I read the status register as well, but it's not done anymore.